src/foam/mlang/MLang.java

```diff
@@ -28,6 +29,8 @@
   public static final Predicate TRUE  = new True();
   public static final Predicate FALSE = new False();
 
+  public static final Date EOD = new Date(( System.currentTimeMillis() / ( 3600 * 24 * 1000 ) + 1 ) * ( 3600 * 24 * 1000 ));
```

Comment:
It should set the value to "new EOD()", or EOD.instance(), if it's a singleton.;

Reply:
We don't have singleton for java. 
Needs to be manually added. 
Example from IPSupport  
```
axioms: [
    {
      name: 'javaExtras',
      buildJavaClass: function(cls) {
        cls.extras.push(foam.java.Code.create({
          data: `
  private final static IPSupport instance__ = new IPSupport();
  public static IPSupport instance() { return instance__; }
          `
        }));
      }
    }
  ],

```